cypress/e2e/Providers.spec.cy.ts

```diff
@@ -0,0 +1,8 @@
+describe("Providers screen", () => {
+  it("opens create dialog", () => {
+    cy.visit("/");
+    cy.contains(".pf-c-nav__link", "Migration").click();
+    cy.contains(".pf-c-nav__link", "Providers for virtualization").click();
+    cy.contains('button[type="button"]', "Create Provider").click();
```

Comment:
The actual tests are a different issue, this will need it's own tasks:

a. add test ID https://testing-library.com/docs/queries/bytestid/
b. separate test to tiers, run only "tier 0" / "smoke tests" for each PR, ( run full tests once a week ?)

the actually tests and testIDs will be addressed in a different task "create e2e smoke tests"

Reply:
right. this test should be actually named  `it("clicks Create Provider button")` :)